test/profiler-mocha.js

```diff
@@ -172,6 +172,11 @@ describe("profiler", function() {
                nodeInstance.profiler.stopEvent.should.be.a('function');
                nodeInstance.profiler.startUpdate.should.be.a('function');
                nodeInstance.profiler.stopUpdate.should.be.a('function');
+               function delay(ms) {
+                  ms += new Date().getTime();
+                  while (new Date() < ms){}
+               }
+               delay(1); // wait long enough to ensure average update time is greater than 0
```

Comment:
Can this be changed to use common-stubs.js#promiseLater?
https://github.com/rdf-pipeline/noflo-rdf-components/blob/master/test/common-stubs.js#L31